WordPress/Classes/ViewRelated/Views/AlertView/AlertInternalView.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+
+
+/**
+*  @class           AlertInternalView
+*  @brief           Helper class, used internally by AlertView. Not designed for general usage.
+*/
+
+public class AlertInternalView : UIView
+{
+    // MARK: - Public Properties
+    public var onClick : (() -> ())?
+    
+    
+    
+    // MARK: - View Methods
+    public override func awakeFromNib() {
+         super.awakeFromNib()
+        
+        // Check the Outlets
+        assert(backgroundView   != nil)
+        assert(alertView        != nil)
+        assert(titleLabel       != nil)
+        assert(descriptionLabel != nil)
+
+        // Setup please!
+        alertView.layer.cornerRadius    = cornerRadius
+        titleLabel.font                 = Styles.titleRegularFont
+        descriptionLabel.font           = Styles.detailsRegularFont
+
+        titleLabel.textColor            = Styles.titleColor
+        descriptionLabel.textColor      = Styles.detailsColor
+
+        dismissButton.titleLabel?.font  = Styles.buttonFont
```

Comment:
Old habits die hard! i'll remove that, thanks Diego!. Hooowever, let's keep the alignment for the outlets. Improves readability a whoole lot!